inputs.yaml

```diff
@@ -0,0 +1,6 @@
+#File for setting Snakefile parameters
+notebook: ttbar_analysis_reana.ipynb
```

Comment:
Hmm, the notebook input could be confusing... We used it elsewhere when the container image is generic (and could run _any_ ROOT macro for example), but here the demo example is pretty specific, and chances are that nobody would like to run another notebook using this image, or introduce a new input file, etc.

Hence, for the clarity, we could keep only real "parameters" in this file: 

- N_FILES_MAX_PER_SAMPLE
- USE_DASK

And leave the rest hard-coded in the notebooks.